staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/install/install.go

```diff
@@ -27,5 +30,6 @@ import (
 func Install(scheme *runtime.Scheme) {
 	utilruntime.Must(apiextensions.AddToScheme(scheme))
 	utilruntime.Must(v1beta1.AddToScheme(scheme))
-	utilruntime.Must(scheme.SetVersionPriority(v1beta1.SchemeGroupVersion))
+	utilruntime.Must(v1.AddToScheme(scheme))
+	utilruntime.Must(scheme.SetVersionPriority(v1beta1.SchemeGroupVersion, v1.SchemeGroupVersion))
```

Comment:
Is it intentional that v1beta1 has a higher priority than v1? I found this setting is different from the community config in https://github.com/kubernetes/kubernetes/blob/cea1d4e20b4a7886d8ff65f34c6d4f95efcb4742/staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/install/install.go.

Reply:
yes, you are correct. Community has dedicated PR to prefer CRD V1: https://github.com/kubernetes/kubernetes/pull/84276. In order to reduce the complication of this PR, so not included community PR#84276 in this PR. we can think about if necessary in the future.